db/migrate/20170620222146_change_games_columnlast_move_pawn_two_steps_second_time.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeGamesColumnlastMovePawnTwoStepsSecondTime < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :games, :last_move_en_passant_situation?, :piece_capturable_by_en_passant
+    change_column :games, :piece_capturable_by_en_passant, :string
```

Comment:
Since this is being used to store an ID.

Could set this column to be an integer (integers can be null btw, so it will work for your case where you check `blank?`).

And you could have it be `piece_capturable_by_en_passant_id` And take advantage of Rails associations.